### PR TITLE
Needed media options so hard...

### PR DESCRIPTION
**Pull request by mkcoldwolf**
Media Options are kinda important feature for other plugins to integrate with 
especially the setLoop for Spatial sound 
as it seems a unfinished feature, hope it can be implemented soon

### CODE REVIEW

File: plugin/src/main/java/com/craftmend/openaudiomc/generic/media/objects/MediaOptions.java [hunk 1]
```diff
@@ -13,38 +13,94 @@
...
-    }
+	private boolean loop = false;
+	private String id;
+	private int expirationTimeout = -1;
+	private boolean pickUp = true;
+	private int fadeTime = 0;
+	private int volume = 100;
+	private List<AudioNode> effectNodes = new LinkedList<>();
+
+	public boolean isLoop() {
```
Comment (Mindgamesnl) [inline plugin/src/main/java/com/craftmend/openaudiomc/generic/media/objects/MediaOptions.java:24]: OA uses Lombok for getters and setters

File: plugin/src/main/java/com/craftmend/openaudiomc/generic/media/objects/OptionalError.java [hunk 2]
```diff
@@ -7,7 +7,14 @@
 @AllArgsConstructor
 public class OptionalError {
 
-    private boolean error;
-    private String message;
+	private boolean error;
+	private String message;
+
+	public OptionalError(boolean error, String message) {
```
Comment (Mindgamesnl) [inline plugin/src/main/java/com/craftmend/openaudiomc/generic/media/objects/OptionalError.java:13]: Already covered by Lombok

**Comment by Mindgamesnl** [comment]
Are you gonna finish this? If so;
 - Please check the linked notes, especially those related to lombok and annotation usage.
 - Please check the client implementation, media options were originally implemented as an internal API between platforms but were later exposed to the end-user, so this will probably collide at some point.

Or please close the PR and just make an issue (or drop it in the discord) if you're not planning on finishing it yourself.

**Comment by mkcoldwolf** [comment]
Ah, i see, didn't notices OA use Lombok for getter setter
Sorry for making troubles here, will just open an issue instead

**Pull request closed without merging**